geometry_msgs/Pose2D global_pose
int64 lane_id
int64 action

